src/documents/css/idg.css.styl

```diff
@@ -434,9 +434,78 @@ setActiveCategory(inputColor) {
     list-style-type: none;
 }
 
+/* Footer logos */
+
+.footer-logo-links svg {
+    width: 6rem;
+    height: 7rem;
+    fill: #fff;
+    stroke: header-footer-background;   
+}   
+
+/* remove hover and focus from the anchors (override template), apply to svgs only */
+a:hover.footer-logo-links { 
+    background: none; 
+}
+
+a:hover.footer-logo-links svg {
+    fill: header-footer-background;
+    background-color: #FFF;
+}
+
+a:focus.footer-logo-links {
+    outline: 0;
+}
+
+a:focus.footer-logo-links svg {
+    outline: header-footer-focus;
+}
+
+/* remove default link underline from svg logos */
+a.footer-logo-links {
+    text-decoration: none;
+} 
+
+/* Special styling for P4A Logo. The tree and "Prosperity" text should
+appear inverted in order to look proper. */
+#tree, #prosperity {
+    stroke: header-footer-background;
+    fill: header-footer-background;
+}
+
+a:hover #tree, a:hover #prosperity {
+    stroke: header-footer-background;
+    fill: #fff;
+}
+
 /* Contrast Themes */
 
 contrast-theme(foregroundColor, backgroundColor) {
+  
+/* remove hover and focus from the anchors (override template), apply to svgs only */  
+    a:hover.footer-logo-links {
+        background: none !important;
+    }
+    
+    a:focus.footer-logo-links {
+        outline: 0 !important;
+    }
+    
+    a:hover.footer-logo-links svg {
+        fill: foregroundColor;
```

Comment:
Note 1:
Since we're using Stylus, we can use it to simplify and make CSS easier to read. For example instead of defining two rules:

a:hover.footer-logo-links {
        background: none !important;
}
    
a:hover.footer-logo-links svg {
     fill: foregroundColor;
}

We can write this instead:

a:hover.footer-logo-links {
        background: none !important;
        svg {
                 fill: foregroundColor;
        }
}